src/framework/services/event/EscapeKeyHandler.spec.js

```diff
@@ -0,0 +1,61 @@
+
+import EscapeKeyHandler from './EscapeKeyHandler';
+
+describe('EscapeKeyHandler', () => {
+  describe('constructor', () => {
+    it('sets callback of instance to passed method', () => {
+      const callback = () => undefined;
```

Comment:
Our code style doesn't allow this (http://eslint.org/docs/rules/no-empty-function).